src/libcore/char/mod.rs

```diff
@@ -45,13 +45,13 @@ use crate::fmt::{self, Write};
 use crate::iter::FusedIterator;
 
 // UTF-8 ranges and tags for encoding characters
-const TAG_CONT: u8     = 0b1000_0000;
-const TAG_TWO_B: u8    = 0b1100_0000;
-const TAG_THREE_B: u8  = 0b1110_0000;
-const TAG_FOUR_B: u8   = 0b1111_0000;
-const MAX_ONE_B: u32   =     0x80;
-const MAX_TWO_B: u32   =    0x800;
-const MAX_THREE_B: u32 =  0x10000;
+const TAG_CONT: u8 = 0b1000_0000;
+const TAG_TWO_B: u8 = 0b1100_0000;
+const TAG_THREE_B: u8 = 0b1110_0000;
+const TAG_FOUR_B: u8 = 0b1111_0000;
+const MAX_ONE_B: u32 = 0x80;
+const MAX_TWO_B: u32 = 0x800;
+const MAX_THREE_B: u32 = 0x10000;
```

Comment:
Is there a way to register exceptions from rustfmt? The manual alignment here, at least for the first tags that are written in binary, was quite useful IMO.

Reply:
There is, but I don't think the alignment is enough of a readability improvement to be worth making an exception here. I think the underscore grouping is sufficient to read them clearly. I am amenable in general to making rustfmt exceptions but I think it doesn't make enough of a difference in this case.